### PR TITLE
ENG-2024 DataTable component - amend row bulk selection and column sorting options

### PR DESCRIPTION
**Pull request by jeffgo10**
+ package.json of ddtable update for unit testing

### CODE REVIEW

File: packages/entando-datatable/test/DataTable.test.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import '@testing-library/jest-dom/extend-expect';
-import { render } from '@testing-library/react';
+import { render, fireEvent, screen, createEvent, debug } from '@testing-library/react';
```
Comment (gidesan) [inline packages/entando-datatable/test/DataTable.test.js:3]: please use `user-events` library instead of direct call to `fireEvents` when possible

**Review by gidesan — CHANGES_REQUESTED** [review]

File: packages/entando-datatable/test/DataTable.test.js (hunk 1 above)
Comment (jeffgo10) [reply packages/entando-datatable/test/DataTable.test.js:3]: Hi @gidesan . There aren't any drag and drop events on `user-events` as I recalled on us discussing this in a related previous task. Though I can implement both if you want (use `fireEvent` on DnD-related functions, `user-events` on other events). Let me know what you think.

Comment (jeffgo10) [reply packages/entando-datatable/test/DataTable.test.js]: Hi @gidesan I have made changes. pls review

**Review by gidesan — APPROVED** [review]
I'll approve this one but tbh I'm quite worried about how big the component is getting. I'll schedule a deeper analysis of this component.

**Comment by jeffgo10** [comment]
no problem @gidesan . Let me know when you found something :)

**Pull request merged**